src/App.js

```diff
@@ -1,7 +1,9 @@
+import Calculator from './components/Calculator';
+
 function App() {
   return (
     <div className="wrapper">
-      Hello World!
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] Please use the class-based component for the app component as it is the requirement for the project that all components should be class-based components.

### See the image of the requirement

![image](https://user-images.githubusercontent.com/83803414/165086297-2327afef-32dd-4cff-8d2b-529d24e062ad.png)